common/config/base.php

```diff
@@ -60,7 +60,7 @@
             'username' => env('DB_USERNAME'),
             'password' => env('DB_PASSWORD'),
             'tablePrefix' => env('DB_TABLE_PREFIX'),
-            'charset' => 'utf8',
+            'charset' => (env('DB_CHARSET') ? env('DB_CHARSET') : 'utf8'),
```

Comment:
i don't see much probability that charset may differ in different environments. Do we really need this?